src/Car.js

```diff
@@ -0,0 +1,26 @@
+import constants from './constants/constants';
+import { Random } from '@woowacourse/mission-utils';
+
+class Car {
+  constructor(name) {
+    this.name = name;
+    this.moveCount = 0;
+  }
+
+  move = () => {
+    const randomNumber = Random.pickNumberInRange(
+      constants.NUMBER_RANGE_MIN,
+      constants.NUMBER_RANGE_MAX,
+    );
+
+    if (randomNumber >= constants.JUDGE_NUMBER) {
+      this.moveCount++;
+    }
+  };
+
+  printScore = () => {
+    return `${this.name} : ${'-'.repeat(this.moveCount)}`;
+  };
+}
```

Comment:
클래스 필드에 함수를 할당하는 경우, 이 함수는 프로토타입 메서드가 아닌 인스턴스 메서드가 되기 때문에 ( = 객체마다 새롭게 생성 )
클래스 필드에 함수를 할당하는 것은 권장하지 않는다고 알고 있습니다! 🥺
move, printSCore 같은 경우는 인스턴스 메서드일 필요가 없어보이니 ES6 메서드 선언 방식을 이용하면 어떨까요? 

```suggestion
  move () {
    const randomNumber = Random.pickNumberInRange(
      constants.NUMBER_RANGE_MIN,
      constants.NUMBER_RANGE_MAX,
    );

    if (randomNumber >= constants.JUDGE_NUMBER) {
      this.moveCount++;
    }
  };

  printScore() {
    return `${this.name} : ${'-'.repeat(this.moveCount)}`;
  };
}
```